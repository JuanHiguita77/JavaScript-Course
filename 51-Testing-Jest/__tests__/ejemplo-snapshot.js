const cliente =
{
	nombre: 'juan',
	balance: 800,
	tipo: 'premium'
}

describe('Prueba de snapshot',()=>
{
	test('Es Juan Pablo',()=>
	{
		expect(cliente).toMatchSnapshot();
	});
});

//Se puede reescribir los snapshots en la terminal: npm t -- -u