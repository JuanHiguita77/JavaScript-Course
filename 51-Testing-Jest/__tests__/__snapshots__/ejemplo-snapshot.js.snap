// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Prueba de snapshot Es Juan Pablo 1`] = `
{
  "balance": 800,
  "nombre": "juan",
  "tipo": "premium",
}
`;
